metals/src/main/scala/scala/meta/internal/metals/MetalsEnrichments.scala

```diff
@@ -393,7 +393,9 @@ object MetalsEnrichments
     def isScalaProject(): Boolean =
       containsProjectFilesSatisfying(_.isScala)
     def isMetalsProject(): Boolean =
-      containsProjectFilesSatisfying(_.isScalaOrJavaFilename)
+      path.resolve(".metals").exists ||
+        path.resolve(".bloop").exists ||
```

Comment:
Should we also include `.bsp` ?

Reply:
It's a very Scala thing, so I added it too.